### PR TITLE
Feature max output file size

### PR DESCRIPTION
**Pull request by ikenchina**

### CODE REVIEW

File: output_file.go [hunk 1]
```diff
@@ -16,6 +17,11 @@ import (
 	"time"
 )
 
+var (
+	// output bytes reach max size
```
Comment (ikenchina) [inline output_file.go]: fixed, thanks

**Comment by buger** [comment]
We already have such option called `--exit-after`. For example: `goreplay --exit-after 5m ...`. 

Can you describe a bit your use case, and if `--exit-after` flag is not enough for you.

Thank you!

**Comment by ikenchina** [comment]
--exit-after is designed for time.
this parameter for size of output file.
i want to record traffics but don't want run out of my disk

**Comment by buger** [comment]
manually merged to master

**Pull request closed without merging**